lib/src/push_notifications/src/remote_message.dart

```diff
@@ -8,20 +8,23 @@ import 'package:ably_flutter/src/platform/platform_internal.dart';
 /// See https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/RemoteMessage
 /// See https://developer.apple.com/documentation/usernotifications/unnotificationcontent
 class RemoteMessage {
-  /// Data part of notification, from custom payload
+  /// Data part of notification, from custom payload.
+  ///
   /// Comes from 'RemoteMessage.data' on Android and
   /// 'UNNotificationContent.userInfo' on iOS
   Map<String, dynamic> data;
 
   /// Notification part of push message
   Notification? notification;
 
+  /// @nodoc
   /// Initializes an instance with [data] set to empty map
   RemoteMessage({
     Map<String, dynamic>? data,
     this.notification,
   }) : data = data ??= {};
 
+  /// @nodoc
   /// create instance from a map
```

Comment:
```suggestion
  /// Create instance from a map.
```

Reply:
Done in 04b14d30b2d52bd3b1c04ca9d12ccc1963a503a2